edb/server/main.py

```diff
@@ -244,18 +245,27 @@ async def _run_server(
             new_instance=new_instance,
             admin_ui=args.admin_ui,
             disable_dynamic_system_config=args.disable_dynamic_system_config,
-            compiler_state=compiler_state,
+            compiler_state=compiler.state,
             tenant=tenant,
             use_monitor_fs=args.reload_config_files in [
                 srvargs.ReloadTrigger.Default,
                 srvargs.ReloadTrigger.FileSystemEvent,
             ],
             net_worker_mode=args.net_worker_mode,
         )
+        magic_smtp = os.getenv('EDGEDB_MAGIC_SMTP_CONFIG')
```

Comment:
Did you do a manual test of this, since apparently I half-assed it and didn't write one

Reply:
Yes, manually tested ✅